ksql-engine/src/test/resources/query-validation-tests/key-field.json

```diff
@@ -112,15 +148,34 @@
       ],
       "post": {
         "issues": [
-          "key field of output has INCORRECT name: should be 'FOO'.",
-          "key field of output has INCORRECT type: should be 'INT"
+          "key field of output has INCORRECT name: should be 'FOO' - (see Issue #1 above)",
+          "key field of output has INCORRECT type: should be 'INT' - (see Issue #3 above)"
         ],
         "sources": [
           {"name": "INPUT", "type": "stream", "keyField": null},
           {"name": "OUTPUT", "type": "table", "keyField": {"name": "KSQL_INTERNAL_COL_0", "schema": {"type": "STRING"}}}
         ]
       }
     },
+    {
+      "name": "stream | initially null | group by (-) | key in value | no aliasing | legacy downstream query",
```

Comment:
I'm not sure I understand what is "legacy" about downstream?

Reply:
When I add a fix for these issues these 'legacy downstream' queries should continue to produce the same, incorrect, state and messages. Thereby proving/testing I've not broken/changed persistent queries running on customers clusters that were started in previous versions of KSQL